src/style.css

```
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
```

you have wrap here, thus you don't need `column` direction
```suggestion
  flex-wrap: wrap;
```

---

Fixed